multiqc/modules/rna_seqc/rna_seqc.py

```diff
@@ -107,15 +108,66 @@ def rnaseqc_general_stats (self):
             'scale': 'YlGn',
             'modify': lambda x: float(x) * 100.0
         }
+        headers['Expression Profiling Efficiency'] = {
```

Comment:
Ok, I don't really understand the difference still but that's fine - could you drop the `% Exonic` as it's in the plot below then please? And maybe add a little more text to the rollover description for anyone else like me who doesn't recognise the term 😄